src/test/java/com/google/alpollo/AnalysisTest.java

```diff
@@ -206,4 +220,60 @@ public void sentimentIsMixed() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void foundArtistWithFullName() throws IOException {
+    when(request.getReader()).thenReturn(
+        new BufferedReader(new StringReader(gson.toJson(FULL_ARTIST_NAME))));
+    artistServletUnderTest.doPost(request, response);
+    String responseString = responseWriter.toString();
+
+    List<String> actual = gson.fromJson(responseString, new TypeToken<List<String>>(){}.getType());
+    List<String> expected = Arrays.asList("Dr. Dre", "Michael Clarke", "Eminem", "Bad Meets Evil", "Vincent Vinel");
+
+
+    Assert.assertEquals(expected, actual);
+  }
+
+  @Test
+  public void foundArtistWithIncompleteName() throws IOException {
+    when(request.getReader()).thenReturn(
+        new BufferedReader(new StringReader(gson.toJson(INCOMPLETE_ARTIST_NAME))));
+    artistServletUnderTest.doPost(request, response);
+    String responseString = responseWriter.toString();
+
+    List<String> actual = gson.fromJson(responseString, new TypeToken<List<String>>(){}.getType());
+    List<String> expected = Arrays.asList("Dr. Dre", "Michael Clarke", "Eminem", "Emine Erdoğan", "Emine Gülbahar Hatun");
+
+
+    Assert.assertEquals(expected, actual);
+  }
+
+  @Test
+  public void foundArtistWithWrongName() throws IOException {
+    when(request.getReader()).thenReturn(
+        new BufferedReader(new StringReader(gson.toJson(WRONG_ARTIST_NAME))));
+    artistServletUnderTest.doPost(request, response);
+    String responseString = responseWriter.toString();
+
+    List<String> actual = gson.fromJson(responseString, new TypeToken<List<String>>(){}.getType());
+    List<String> expected = Arrays.asList();
+
```

Comment:
nit: empty line

Reply:
But I like those empty lines, they separate the initializations from the actual execution and then the execution from the assertion

Reply:
you can have one empty line, of course, but why two :D

Reply:
I see only one empty line between each of the three code blocks.